### DISCUSSION TITLE
how to count table column?

### DISCUSSION THREAD

**Discussion by WingDust** [nushell/nushell#11036] [Q&A] [answered]
this is a csv 
```csv
asd,wasd,wad
2172,2,6
2,11,4
367,7,6
31,8,6
```
 but  I want count `asd` column to get `2572` like functional language `countBy` function

`cat test.csv | from csv `

**Comment by ghost** [comment, marked as answer]
`math sum` will do the job:

```
open my.csv | get asd | math sum
```
